web/src/views/TwitterDownloaderView.vue

```diff
@@ -1,258 +1,249 @@
-                    <form
-                        class="needs-validation"
-                        @submit.prevent>
-                        <div class="row rounded-top ">
-                            <div class="input-group">
-                                <div class="input-group mb-3">
-                                    <input
-                                        type="text"
-                                        class="form-control form-control-lg"
-                                        placeholder="Enter Video Link..."
-                                        aria-label="Enter Video Link..."
-                                        aria-describedby="button-addon2"
-                                        v-model="downloadForm.url"
-                                        :disabled="isLoading"
-                                        :class="{
-                                            'is-invalid': v$.url.$error && v$.url.$dirty,
-                                            'is-valid': !v$.url.$error && v$.url.$dirty
-                                        }"
-                                        @blur="v$.url.$touch" />
-
-                                    <div
-                                        v-for="error of v$.url.$errors"
-                                        class="invalid-feedback"
-                                        :key="error.$uid">
-                                        {{ error.$message }}
-                                    </div>
-                                </div>
-                            </div>
-                        </div>
-                    </form>
-
-                    <button
-                        id="button-addon2"
-                        class="btn btn-primary"
-                        type="button"
-                        :disabled="isLoading || v$.$invalid"
-                        @click="getTweetMedia()">
-                        <span v-if="!isLoading">
-                            Download
-                        </span>
-
-                        <span v-else>
-                            <span
-                                class="spinner-grow spinner-grow-sm"
-                                role="status"
-                                aria-hidden="true" />
-                            Loading...
-                        </span>
-                    </button>
-
-                    <figure class="text-center mt-4">
-                        <figcaption class="blockquote-footer">
-                            Link example:
-                            https://twitter.com/unicornify/status/882987478541533189
-                        </figcaption>
-                    </figure>
-                </div>
-            </div>
-        </div>
-    </div>
+					<figure class="text-center mt-4">
+						<figcaption class="blockquote-footer">
+							Link example:
+							https://twitter.com/unicornify/status/882987478541533189
+						</figcaption>
+					</figure>
```

Comment:
It would be better if we put the example as a placeholder for TheInput. example:
`Enter link (Eg: https://xxx/xxx)`